kod/object/active/portal/newbport.kod

```diff
@@ -90,7 +90,9 @@ messages:
             Send(what,@AdminGotoSafety);
          }
 
-         Send(what,@AddExertion,#amount=(send(what,@GetVigor) - 80) * 10000);
+         % Nearly everyone will be leaving Raza at around 200 vigor after
+         % hitting 25, so top them off before sending them on their way.
+         Send(what, @EatSomething, #nutrition=200);
```

Comment:
Out of curiosity, why `EatSomething` over `AddExertion`? Less convoluted? And it looks like not passing in `filling` means they get 200 vigor w/o the fullness effect?

Reply:
Yeah, less convoluted and easier to look at since the nutrition value and vigor are 1:1.  And yeah, no filling value should default it to 0.  There was at least one other thing in the game that used the same approach which is where I got the idea, but I can't remember where I saw it exactly.  If addexertion is preferred, it's easy enough for me to switch it to that.

Reply:
I'm not 100% convinced this is safe, given that it changes the function that is called, and the existing one is known to work.  Plus I'm not sure this is really much of a problem, so the risk outweighs the reward for me.  If you use AddExertion instead it will likely look like a safer change.